feature/experimental/bgp/ate_tests/bgp_remove_private_as/bgp_remove_private_as_test.go

```diff
@@ -0,0 +1,347 @@
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	ateSrc = attrs.Attributes{
+		Name:    "ateSrc",
+		IPv4:    "192.0.2.2",
+		IPv6:    "2001:db8::192:0:2:2",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	dutDst = attrs.Attributes{
+		Desc:    "DUT to ATE destination",
+		IPv4:    "192.0.2.5",
+		IPv6:    "2001:db8::192:0:2:5",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+	ateDst = attrs.Attributes{
+		Name:    "atedst",
+		IPv4:    "192.0.2.6",
+		IPv6:    "2001:db8::192:0:2:6",
+		IPv4Len: plenIPv4,
+		IPv6Len: plenIPv6,
+	}
+)
+
+// configureDUT configures all the interfaces on the DUT.
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
+	dc := gnmi.OC()
+	i1 := dutSrc.NewOCInterface(dut.Port(t, "port1").Name(), dut)
+	gnmi.Replace(t, dut, dc.Interface(i1.GetName()).Config(), i1)
+
+	i2 := dutDst.NewOCInterface(dut.Port(t, "port2").Name(), dut)
+	gnmi.Replace(t, dut, dc.Interface(i2.GetName()).Config(), i2)
+}
+
+// verifyPortsUp asserts that each port on the device is operating.
+func verifyPortsUp(t *testing.T, dev *ondatra.Device) {
+	t.Helper()
+	for _, p := range dev.Ports() {
+		status := gnmi.Get(t, dev, gnmi.OC().Interface(p.Name()).OperStatus().State())
+		if want := oc.Interface_OperStatus_UP; status != want {
+			t.Errorf("%s Status: got %v, want %v", p, status, want)
+		}
+	}
+}
+
+// bgpCreateNbr creates a BGP object with neighbors pointing to ateSrc and ateDst.
+func bgpCreateNbr(localAs, peerAs uint32, dut *ondatra.DUTDevice) *oc.NetworkInstance_Protocol {
+	nbr1v4 := &bgpNeighbor{as: ateAS1, neighborip: ateSrc.IPv4, isV4: true, peerGrp: peerGrpName1}
+	nbr2v4 := &bgpNeighbor{as: ateAS2, neighborip: ateDst.IPv4, isV4: true, peerGrp: peerGrpName2}
+	nbrs := []*bgpNeighbor{nbr1v4, nbr2v4}
+
+	d := &oc.Root{}
+	ni1 := d.GetOrCreateNetworkInstance(deviations.DefaultNetworkInstance(dut))
+	ni_proto := ni1.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
+	bgp := ni_proto.GetOrCreateBgp()
+	global := bgp.GetOrCreateGlobal()
+	global.RouterId = ygot.String(dutDst.IPv4)
+	global.As = ygot.Uint32(localAs)
+	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
+
+	// Note: we have to define the peer group even if we aren't setting any policy because it's
+	// invalid OC for the neighbor to be part of a peer group that doesn't exist.
+	pg1 := bgp.GetOrCreatePeerGroup(peerGrpName1)
+	pg1.PeerAs = ygot.Uint32(ateAS1)
+	pg1.PeerGroupName = ygot.String(peerGrpName1)
+
+	pg2 := bgp.GetOrCreatePeerGroup(peerGrpName2)
+	pg2.PeerAs = ygot.Uint32(ateAS2)
+	pg2.PeerGroupName = ygot.String(peerGrpName2)
+
+	for _, nbr := range nbrs {
+		nv4 := bgp.GetOrCreateNeighbor(nbr.neighborip)
+		nv4.PeerGroup = ygot.String(nbr.peerGrp)
+		nv4.PeerAs = ygot.Uint32(nbr.as)
+		nv4.Enabled = ygot.Bool(true)
+		af4 := nv4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST)
+		af4.Enabled = ygot.Bool(true)
+		af6 := nv4.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST)
```

Comment:
if we are using afisafi oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST at neighbor-level, then we need to enable it at global level. Something similar to line '129'

Reply:
updated, thanks.

Reply:
Hi @cprabha , were below two lines removed to address this comment ?
These are actually needed to enable AF at BGP global level, could you add them back.

	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).Enabled = ygot.Bool(true)
	global.GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).Enabled = ygot.Bool(true)

Reply:
@arvbaska1 , we have enabled at neighbor level, neighbor level takes the precedence than global level.

Reply:
Global AF enable is needed for us to enable the neighbor level address-family. 
All the other BGP tests have the same configuration.So could you add them here as well ?

Reply:
@sachendras @arvbaska1 , updated please check

Reply:
Thanks, looks good.